Docker usage throws this error: Cannot find module '/src/bin.js'

```
$ alias pin-github-action="docker run --rm -v $(pwd):/src -e GH_ADMIN_TOKEN mheap/pin-github-action"
$ export GH_ADMIN_TOKEN=REDACTED
$ pin-github-action .github/workflows/docker-build-push.yml
Unable to find image 'mheap/pin-github-action:latest' locally
latest: Pulling from mheap/pin-github-action
1f3e46996e29: Pull complete 
11bcb6612b10: Pull complete 
e9a6e3e876c0: Pull complete 
40f144496201: Pull complete 
3628a6a42627: Pull complete 
5d07909e909f: Pull complete 
c52d65f7aaa6: Pull complete 
218d4db0784f: Pull complete 
Digest: sha256:beeb5d619eff9efbdd8e212f9c3a97c657dba80302d0f5aac37d5b8956d6b5bb
Status: Downloaded newer image for mheap/pin-github-action:latest
node:internal/modules/cjs/loader:1228
  throw err;
  ^

Error: Cannot find module '/src/bin.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:173:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.18.3
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -13,7 +13,9 @@ RUN npm install --production
 # Copy the application source code
 COPY . .
 
+WORKDIR /workflows
+
 # Set the entry point for the container to run the CLI
 # Replace 'your-cli-command' with the actual command of your CLI app
-ENTRYPOINT ["node", "bin.js"]
+ENTRYPOINT ["node", "/src/bin.js"]
 

--- a/README.md
+++ b/README.md
@@ -57,7 +57,7 @@ npm install -g pin-github-action
 ### Docker
 
 ```bash
-alias pin-github-action="docker run --rm -v $(pwd):/src -e GITHUB_TOKEN mheap/pin-github-action"
+alias pin-github-action="docker run --rm -v $(pwd):/workflows -e GITHUB_TOKEN mheap/pin-github-action"
 ```
 
 ## Usage

